mavros_extras/src/plugins/odom.cpp

```diff
@@ -26,21 +25,24 @@ namespace extra_plugins {
  * @brief Odometry plugin
  *
  * Send odometry info
- * to FCU position and attitude estimators.
- *
+ * to FCU position and attitude estimators
  */
 class OdometryPlugin : public plugin::PluginBase {
 public:
 	OdometryPlugin() : PluginBase(),
-		_nh("~odometry")
-	{}
+		odom_nh("~odometry"),
+		estimator_type(3)
+	{ }
 
 	void initialize(UAS &uas_)
 	{
 		PluginBase::initialize(uas_);
 
-		// tf params
-		_odom_sub = _nh.subscribe("odom", 10, &OdometryPlugin::odom_cb, this);
+		// general params
+		odom_nh.param("estimator_type", estimator_type, 3); // defaulted to VIO type
```

Comment:
I kept it as `int` so it can be easily received and modified on the `yaml` file.